app/form-components/team-form/TeamMembersForm.tsx

```diff
@@ -104,6 +104,17 @@ export default function TeamForm({ errors, members, setMembers, allowDelete = tr
         <strong>2 business days</strong>.
       </p>
       <br />
+
+      <p>
+        <strong>Roles:</strong>
+      </p>
+      <p>
+        <u>Admin</u>: can manage integrations <strong>and</strong> teams
```

Comment:
To underline text, it is preferable to use the CSS text-decoration property set to underline.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u

To bold text, it is preferable to use`text-weight` CSS
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#usage_notes